index.html

```diff
@@ -241,5 +256,50 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <section class="form">
+      <p class="form-text">
+        <!-- I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. -->
+      </p>
+      <!-- This is the Form section of the Page with validations -->
+      <form class="form-inputs" action="https://formspree.io/f/xyyoqbgr" method="post">
+        <input type="text" class="form-name" name="name" placeholder="Full name" maxlength="30" required>
+        <input type="email" class="form-email" name="email" placeholder="Email address" required>
+        <textarea name="Message" class="form-comments" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
```

Comment:
- [ ] Kindly make sure all inputs fields stick to the Figma design as required for the project(e.g: check the text padding(left & right), the borders,...). Make sure to check every detail to stick to your chosen template.

Reply:
@tresorsawasawa  the form now marches figma template.Issues fixed